arch/arm/boot/dts/overlays/irs1125-overlay.dts

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for IRS1125 camera module on VC I2C bus
+/dts-v1/;
+/plugin/;
+
+/{
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target = <&i2c_vc>;
+		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+
+			irs1125: irs1125@36 {
+				compatible = "infineon,irs1125";
+				reg = <0x3D>;
+				status = "okay";
+
+				pwdn-gpios = <&expgpio 5 0>;
```

Comment:
The gpio values are going to be overwritten by the firmware so what I'm describing isn't a bug, but it would help my verification script if the default version resolves against all Pi DTBs (it currently won't apply for Pis without a GPIO expander), even if it won't work as written - replacing `&expgpio` with `&gpio` fixes it for me.

Reply:
okay, I changed this